graylog-storage-opensearch2/src/main/java/org/graylog/storage/opensearch2/views/searchtypes/pivot/buckets/OSValuesHandler.java

```diff
@@ -101,6 +106,30 @@ private Script scriptForPivots(Collection<String> pivots) {
         return new Script(scriptSource);
     }
 
+    private TermsAggregationBuilder applyOrdering(Pivot pivot, TermsAggregationBuilder terms, List<String> buckets, List<BucketOrder> ordering, OSGeneratedQueryContext queryContext) {
+        return sortsOnNumericPivotField(pivot, queryContext)
```

Comment:
Do I get it correctly?
Won't we be able to sort on non-numeric pivot fields, like text ones, from now on?

Reply:
Of course we are. The method checks if we are hitting the edge case of sorting on a numeric pivot field and applies a different sorting (adding a sub aggregation and sorting based on it) if that is the case, otherwise it will just apply the ordinary sorting just as before this change.

If you have a suggestion for the code structure that communicates this clearer (it seems like it was not obvious for you when reading the code), then I would love to hear it!

Reply:
The usage of `ordering` variable has disappeared from `createTerms` method and was moved to the last `orElseGet`... Probably that's why I missed it... Thanks for clarification.